src/content/peerconnection/munge-sdp/js/main.js

```diff
@@ -183,10 +185,8 @@ function onCreateSessionDescriptionError(error) {
 function setOffer() {
   var sdp = offerSdpTextarea.value;
   sdp = maybeAddLineBreakToEnd(sdp);
-  var offer = new RTCSessionDescription({
-    type: 'offer',
-    sdp: sdp
-  });
+  sdp = sdp.replace(/\n/g, '\r\n');
```

Comment:
no, just to recreate the CRLF style which is mangled (hah) by the textarea. Now SDP parser are required to deal with both CRLF and LF but I prefer CRLF.